intents/intent.ridl

```diff
@@ -180,13 +186,21 @@ struct TransactionERC1155Value
     + go.field.name = ID
   - amount: string
 
+# NOTE: TransactionDelayedEncode is deprecated, please use TransactionContractCall instead
```

Comment:
I'd change it so that the comment starts with `Deprecated:`, as that will help the IDEs figure it out and better show it to the dev. So something like:
```suggestion
# Deprecated: TransactionDelayedEncode is not type safe, please use TransactionContractCall instead
```